test/e2e/specs/preview.test.js

```diff
@@ -126,41 +75,47 @@ describe( 'Preview', () => {
 		// Title in preview should match input.
 		let previewTitle = await previewPage.$eval( '.entry-title', ( node ) => node.textContent );
 		expect( previewTitle ).toBe( 'Hello World' );
+		await previewPage.close();
 
 		// Return to editor to change title.
 		await editorPage.bringToFront();
 		await editorPage.type( '.editor-post-title__input', '!' );
-		previewPage = await getOpenedPreviewPage();
+		previewPage = await openPreviewPage( editorPage );
 
 		// Title in preview should match updated input.
 		previewTitle = await previewPage.$eval( '.entry-title', ( node ) => node.textContent );
 		expect( previewTitle ).toBe( 'Hello World!' );
+		await previewPage.close();
 
 		// Pressing preview without changes should bring same preview window to
 		// front and reload, but should not show interstitial.
 		await editorPage.bringToFront();
-		previewPage = await getOpenedPreviewPage();
+		previewPage = await openPreviewPage( editorPage );
 		previewTitle = await previewPage.$eval( '.entry-title', ( node ) => node.textContent );
 		expect( previewTitle ).toBe( 'Hello World!' );
+		await previewPage.close();
 
 		// Preview for published post (no unsaved changes) directs to canonical
 		// URL for post.
 		await editorPage.bringToFront();
 		await publishPost();
 		await Promise.all( [
-			page.waitForFunction( () => ! document.querySelector( '.editor-post-preview' ) ),
-			page.click( '.editor-post-publish-panel__header button' ),
+			editorPage.waitForFunction( () => ! document.querySelector( '.editor-post-preview' ) ),
+			editorPage.click( '.editor-post-publish-panel__header button' ),
 		] );
 		expectedPreviewURL = await editorPage.$eval( '.notice-success a', ( node ) => node.href );
-		previewPage = await getOpenedPreviewPage();
+		previewPage = await openPreviewPage( editorPage );
 		expect( previewPage.url() ).toBe( expectedPreviewURL );
 
 		// Return to editor to change title.
 		await editorPage.bringToFront();
 		await editorPage.type( '.editor-post-title__input', ' And more.' );
 
 		// Published preview should reuse same popup frame.
-		previewPage = await getOpenedPreviewPage();
+		// TODO: Fix an existing bug which opens a new tab.
```

Comment:
Can we create an issue to track this work?

Reply:
#8330 - thanks :)
Issue is filed: #8318.